Automattic/jetpack
# My Jetpack: Remove search dependency

Incorporates Search check for required plan into My Jetpack and remove the dependency on the Search package.

Search package is still under development and the package might change. Also, adding this dependency now would increase the bundle size. Let's keep it simple for this MVP.

#### Changes proposed in this Pull Request:

* Incorporates Search check for required plan into My Jetpack and remove the dependency on the Search package.

#### Jetpack product discussion


https://github.com/Automattic/jetpack/pull/22682#issuecomment-1031227045

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:







*    Go to My Jetpack
*    Inspect the global myJetpackInitialState var
*    Verify that all products have has_required_plan equals true except from Search
*   Buy a plan or product that include Search
*    Check again and verify that Search is now also true
